admin/src/templates/settings.html

```diff
@@ -1,9 +1,26 @@
-<ul class="nav nav-tabs" mm-auth="can_configure">
-  <li ui-sref-active="active">
-    <a ui-sref="settings.basic" translate>Basic</a>
-  </li>
-  <li ui-sref-active="active">
-    <a ui-sref="settings.advanced" translate>Advanced</a>
-  </li>
-</ul>
-<div class="tab-content" ui-view="tab" mm-auth="can_configure"></div>
+<div class="tab-content">
+    <form role="form">
+
+        <div class="form-group">
+          <label for="locale" translate>Default Application Language</label>
+          <select id="locale" class="form-control small" ng-model="basicSettingsModel.locale" ng-options="locale.code as locale.name for locale in enabledLocales">
+          </select>
+          <p class="help-block" translate>setup.language.subtitle</p>
+        </div>
+      
+        <div class="form-group">
+          <label for="locale-outgoing" translate>Language For Outgoing Messages</label>
+          <select id="locale-outgoing" class="form-control small" ng-model="basicSettingsModel.locale_outgoing" ng-options="locale.code as locale.name for locale in enabledLocales">
+          </select>
+          <p class="help-block" translate>setup.language.outgoing.subtitle</p>
+        </div>
+      
+        <div class="form-actions">
+          <button type="submit" class="btn btn-primary" ng-click="submitBasicSettings()" ng-disabled="status.loading" translate>Submit</button>
+          <span ng-if="status.success" class="success">{{status.msg}}</span>
+          <span ng-if="status.error" class="error">{{status.msg}}</span>
+          <div class="loader inline" ng-show="status.loading"></div>
+        </div>
+      
+      </form>
+</div>
```

Comment:
The indentation is a little off here. Indent everything by 2 spaces so it's consistent with the other code.